### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by localden**

### CODE REVIEW

File: msal-dotnet-articles/advanced/managed-identity.md [hunk 1]
```diff
@@ -1,51 +1,79 @@
...
->This is an experimental feature for 1st party use only.
+>This feature is experimental and available from [MSAL.NET](https://www.nuget.org/packages/Microsoft.Identity.Client/) version 4.51.0.
 
-## Managed Identity for Confidential Client
+A common challenge for developers is the management of secrets, credentials, certificates, and keys used to secure communication between services. [Managed identities](/azure/active-directory/managed-identities-azure-resources/overview) in Azure eliminate the need for developers to manage these credentials manually. MSAL.NET supports acquiring tokens through the managed identity capability when used with applications running inside Azure infrastructure, such as:
 
-A common challenge for developers is the management of secrets, credentials, certificates, and keys used to secure communication between services. Managed identities eliminate the need for developers to manage these credentials.
+* [Azure App Service](https://azure.microsoft.com/products/app-service/) (API version `2019-08-01` and above)
+* [Azure VMs](https://azure.microsoft.com/free/virtual-machines/)
+* [Azure Arc](/azure/azure-arc/overview)
```
Comment (gladjohn) [inline msal-dotnet-articles/advanced/managed-identity.md:15]: Should these be pointing to azure docs?

File: msal-dotnet-articles/advanced/managed-identity.md [hunk 2]
```diff
@@ -1,51 +1,79 @@
...
 
-* Cloud Shell
-* Service Fabric
+#### `MsalServiceException` Error Code: `managed_identity_failed_response` Error Message: An unexpected error occurred while fetching the AAD token
 
-## Common Exceptions
+This exception might mean that the resource you are trying to acquire a token for is either not supported or is provided using the wrong resource ID format. Examples of correct resource ID formats include `https://management.azure.com/.default`, `https://management.azure.com`, and `https://graph.microsoft.com`.
 
-### MsalServiceException Error Code: managed_identity_failed_response Error Message: An unexpected error occurred while fetching the Azure AD token
+#### `System.Net.Http.HttpRequestException`: A socket operation was attempted to an unreachable network
```
Comment (gladjohn) [inline msal-dotnet-articles/advanced/managed-identity.md:77]: @neha-bhargava can this also happen if Managed Identity is not turned on in a VM?

File: msal-dotnet-articles/advanced/backup-authentication-system.md [hunk 3]
```diff
@@ -1,26 +1,28 @@
 ---
 title: Backup authentication system
```
Comment (pmaytak) [inline msal-dotnet-articles/advanced/backup-authentication-system.md:2]: Maybe this page is a good candidate to move into a more general resiliency page, like High Availability, as a section.

File: msal-dotnet-articles/advanced/backup-authentication-system.md [hunk 4]
```diff
@@ -1,26 +1,28 @@
...
 ---
 
 # Backup authentication system
 
-The Azure AD backup authentication system enables the caching of credentials process by ESTS in order to provide resiliency during outages in Azure AD's authentication services. In order to assist in speeding up the token retrieval from the backup authentication system, MSAL will provide a routing hint in the form of a header or an extra query parameter in authentication requests sent to ESTS. MSAL will attempt to do this for most of the authentication scenarios but there will come a situation where MSAL is not able to provide this hint due to the absence of user data. However, this can be resolved by the use of `WithCcsRoutingHint(string userObjectIdentifier, string tenantIdentifier)` or `WithCcsRoutingHint(string userName)`.
+The Azure Active Directory backup authentication system enables the caching of credentials processed by the Evolved Security Token Service (ESTS) in order to provide resiliency during outages in Azure AD's authentication services.
```
Comment (pmaytak) [inline msal-dotnet-articles/advanced/backup-authentication-system.md:8]: If I remember, the docs folks didn't want to use ESTS term - only Azure AD (too technical probably?)

```suggestion
The Azure Active Directory has a backup authentication system that enables the caching of credentials in order to provide resiliency during outages in Azure AD's authentication services.
```

File: msal-dotnet-articles/advanced/managed-identity.md (hunk 1 above)
Comment (localden) [reply msal-dotnet-articles/advanced/managed-identity.md:15]: @gladjohn will move that to a separate PR to address.

File: msal-dotnet-articles/advanced/backup-authentication-system.md (hunk 3 above)
Comment (localden) [reply msal-dotnet-articles/advanced/backup-authentication-system.md:2]: Indeed, but I will defer that to another pull request rather than this one.

**Pull request merged**